src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,2 +1,11 @@
-// Add the required props
-export const TodoInfo = () => <>TodoInfo markup</>;
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article
+    className={`TodoInfo ${todo.completed === true ? 'TodoInfo--completed' : ''}`}
```

Comment:
Usee the classnames library for add classes with condition
```suggestion
    className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}
```